### PR TITLE
Allow 0s as a value to the jitter-uniform option.

### LINKED ISSUE #513: Accept 0s as an argument to --jitter-uniform
In the options proto we enforce that the uniform jitter option can't be set to 0:
https://github.com/envoyproxy/nighthawk/blob/8667f7f7adf9c4fbed5872de5585e75e1c93e5b7/api/client/options.proto#L183

There are times when being able to do so is convenient, and this should be allowed but result in a no-op.
AFAICT this would just work when we tweak the validation to allow it.

Context: https://github.com/istio/tools/pull/1190#discussion_r484168703

### PR DESCRIPTION
**Pull request by oschaaf**
Fixes #513

Signed-off-by: Otto van der Schaaf <oschaaf@we-amp.com>

### CODE REVIEW

File: source/client/output_formatter_impl.cc [hunk 1]
```diff
@@ -273,7 +273,9 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
   *fortio_output.mutable_requestedduration() = output.options().duration();
   auto actual_duration = getAverageExecutionDuration(output);
   fortio_output.set_actualduration(actual_duration.count());
-  fortio_output.set_jitter(output.options().has_jitter_uniform());
+  fortio_output.set_jitter(output.options().has_jitter_uniform() &&
```
Comment (dubious90) [inline source/client/output_formatter_impl.cc:276]: There's something I'm missing here. As far as I can tell, the jitter in options is basically just being used as a boolean.  If it's there (and now, if it's greater than 0), then we use jitter.  But where does the int value actually come into play?

**Review by dubious90 — COMMENTED** [review]
Looks good. Just one question I think because I'm confused as to how jitter actually works.

File: source/client/output_formatter_impl.cc (hunk 1 above)
Comment (oschaaf) [reply source/client/output_formatter_impl.cc:276]: In Fortio jittering is a bool flag. On our side it's quite different, but we can emulate Fortio's version of it by using a certain configuration (10% of the qps from the top of my head).
So here we are more or less doing the best we can. Setting the right value here will make the Fortio UI display
wether jitter was set.

Comment (oschaaf) [reply source/client/output_formatter_impl.cc:276]: Oh, and what might be confusing here, is that on our side, next to it being set, it must also have a value.

**Review by dubious90 — APPROVED** [review]

**Pull request merged**